packages/w3p/src/types/provider.ts

```diff
@@ -21,7 +22,13 @@ import type {
 } from './provider-event-bus'
 import type { SolanaProvider } from './solana'
 
-export type RawProvider = EthereumProvider | SolanaProvider | NearRawProvider
+export type ProviderInitArgs = WalletConnectInitArgs
+
+export type RawProvider =
+  | EthereumProvider
+  | SolanaProvider
+  | NearRawProvider
+  | ProviderInitArgs
```

Comment:
just add WalletConnectInitArgs here if it possible

Reply:
done